### ISSUE TITLE
Nuget package for the lib

### ISSUE THREAD

**Issue by MiguelAlho** [VerizonDigital/ectoken#7]
Is there any intent in making a NuGet package with the c# lib available?

**Comment by hattan** [comment]
Thanks for suggestion. Our team will take a look and should have it on Nuget.org soon. I'll confirm when it's publicly available.

**Comment by MikeAgostino** [comment]
Hey @hattan, any updates on having this as a Nuget package?

**Comment by alim4** [comment]
I'm interested in the Nuget package as well

**Comment by matthiasvanneste** [comment]
+1 for the NuGet package!

**Comment by hattan** [comment]
Hi All - Sorry for the delay. We added ECToken to Nuget
https://www.nuget.org/packages/ECToken/

Adding support for more frameworks/netstandard soon to the package.

**Comment by corgalore** [comment]
So, should that Nuget pkg work in an ASP.NET project? I'm getting this error:
`You are trying to install this package into a project that targets '.NETFramework,Version=v4.6', but the package does not contain any assembly references or content files that are compatible with that framework.`

**Comment by hattan** [comment]
Yes, it should work in asp.net. The nuget package currently only includes the dlls for .NET 4.6.1 (.NET standard 1.4). We're working to add support for other versions of .net and .net standard very soon. We'll update the thread once they are available.

Thanks for the heads up.

**Comment by corgalore** [comment]
Thanks! Bumping the Framework to 4.6.1 allowed me to install the Nuget. The Azure App Service is configured for 4.7, so it should run okay up there, too.
